pkgs/development/python-modules/jsonlines/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchPypi, six
+, flake8, pep8-naming, pytest, pytestcov, pytestpep8, tox }:
+
+buildPythonPackage rec {
+  pname = "jsonlines";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "126yxzxq7yqhkd3gxmksxrd5kv10iviq15d4lk464j4xi9cdbf23";
+  };
+
+  propagatedBuildInputs = [ six ];
+
+  checkInputs = [ tox flake8 pep8-naming pytest pytestcov pytestpep8 ];
+  doCheck = false; # tox fails
+  checkPhase = ''
+    tox
```

Comment:
```suggestion
    pytest
```

Reply:
Thanks, a little progress. Now it instead finds no tests and exits with `code 5`:

```
running install tests
/nix/store/afxxp90mqz8mhxxyvf9fjgkx8wx6m18a-python3.7-pep8-1.7.1/lib/python3.7/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
============================= test session starts ==============================
platform linux -- Python 3.7.4, pytest-5.1.0, py-1.8.0, pluggy-0.12.0
rootdir: /build/jsonlines-1.2.0
plugins: cov-2.6.1, pep8-1.0.6
collected 0 items                                                              

============================ no tests ran in 0.01s =============================
builder for '/nix/store/ip3spm2r9bqhg07amb19rasn312kn5ag-python3.7-jsonlines-1.2.0.drv' failed with exit code 5
error: build of '/nix/store/ip3spm2r9bqhg07amb19rasn312kn5ag-python3.7-jsonlines-1.2.0.drv' failed
```

Reply:
Oh wait, I see why, using `pypi` doesn't include the tests in their release. I'll use GitHub instead.

Reply:
Done, I've updated the release @jonringer